compiler/passes/errorHandling.cpp

```diff
@@ -828,6 +841,84 @@ bool ErrorCheckingVisitor::enterCallExpr(CallExpr* node) {
   return true;
 }
 
+class NormalizeTryExprsVisitor : public AstVisitorTraverse {
```

Comment:
I'd define this in a separate file, since it is intended to be used at normalize(), right?

Reply:
yes. New file? Or in normalize?

Reply:
I think new file